### ISSUE TITLE
Status on UI and namespace(space) are not in sync

### ISSUE THREAD

**Issue by mahendrabagul** [loft-sh/loft#187]
As per the message in confirmation box, the pods in the namespace are scaled down to zero; but I see all pods are running in the kube-system namespace even after I put the space to sleep

Confirmation Box
![Screenshot from 2022-02-23 22-11-39](https://user-images.githubusercontent.com/6149190/155365327-8c53b7aa-5b1d-44f2-8efe-c2f8229c0cd2.png)
 
data on UI and status of pods are not matching
![Screenshot from 2022-02-23 22-06-28](https://user-images.githubusercontent.com/6149190/155365110-88c5b424-17df-451b-9eba-a595b06b9128.png)

**Comment by FabianKramm** [comment]
@mahendrabagul I guess the kube system namespace is a little bit special here as those pods do not have a replica set or statefulset as owner. In general its not a good idea as well to let kube system sleeping as it would kill the cluster essentially completely, so this is more or less working as intended

**Comment by mahendrabagul** [comment]
@FabianKramm Yeah agree. However, can we show the same on the UI by some mean? This will just let use know that the kube-system namespace cant be put to sleep. :D

**Comment by FabianKramm** [comment]
@mahendrabagul yeah makes sense, we should do that :D
